chain/network/src/peer_manager/connected_peers.rs

```diff
@@ -29,6 +35,49 @@ pub(crate) struct ConnectedPeer {
     pub throttle_controller: ThrottleController,
     /// Encoding used for communication.
     pub encoding: Option<Encoding>,
+    pub send_accounts_data_demux: demux::Demux<Vec<Arc<SignedAccountData>>, ()>,
+}
+
+impl ConnectedPeer {
+    pub fn send_accounts_data(
+        &self,
+        data: Vec<Arc<SignedAccountData>>,
+    ) -> impl std::future::Future<Output = ()> {
```

Comment:
I'd maybe `use std::future::Future` -- future feels like a vocabulary typed designed for unqalified usage. Super weak opinion.

Reply:
done